plover/assets/main.json

```diff
@@ -25411,7 +25414,10 @@
 "HOPBLG/KEUPB/AOES": "Hodgkin's",
 "HOPBLG/POPBLG": "hodgepodge",
 "HOPBLG/SKWRE/TPHAEUT": "homogeneity",
+"HOPBLS": "honestly",
+"HOPBS": "honest",
```

Comment:
Sorry, that was in my user dictionary. `HO*PBS` is already defined as `honest` in the main dictionary. So that should be fine.